frontend/components/styles/Buttons.tsx

```diff
@@ -66,7 +76,7 @@ export const PostPollToggle = ({
   activeOption: PageType
   setActiveOption: React.Dispatch<React.SetStateAction<PageType>>
 }) => (
-  <Toggle style={{ verticalAlign: 'middle' }}>
+  <div className="flex items-center rounded-full bg-neutral-100 p-1.5 h-10 align-middle border shadow-sm shadow-neutral-50">
```

Comment:
why replace with div?

Reply:
Because Toggle is nothing more than a div wrapper itself. The whole component should be called Toggle tbh

Reply:
The CSS for this frontend repo is very very granular, so every div kind of has its own CSS + class